keyboard/src/modules/Keyboard.rb

```diff
@@ -559,7 +558,7 @@ def SetKeyboard(keyboard)
       # do not try to run this with remote X display
       if Ops.greater_than(Builtins.size(@Apply), 0) && x11_setup_needed
         # Apply cannot be escaped as it is already set of parameters. But it is at least our string and not user provided.
-        @xkb_cmd = "#{File.join(XVersion.binPath, "setxkbmap")} #{@Apply}"
+        @xkb_cmd = "/usr/bin/setxkbmap #{@Apply}"
```

Comment:
Is `@Apply` safe or should we `.shellescape` it?

Reply:
See comment above code I wrote when I do escaping ;)

Reply:
Ah,...:dizzy_face: